Examples/UIExplorer/js/CameraRollView.js

```diff
@@ -94,6 +94,7 @@ var CameraRollView = React.createClass({
         var imageStyle = [styles.image, {width: imageSize, height: imageSize}];
         return (
           <Image
+            key={asset}
```

Comment:
From React docs: "A key is a string": https://facebook.github.io/react/docs/lists-and-keys.html

Could you use the path please just to make this 100% legit?

The other example might be using the key wrong.